test/src/us/kbase/kbasefeaturevalues/test/KBaseFeatureValuesServerTest.java

```diff
@@ -67,6 +67,7 @@
 import us.kbase.auth.ConfigurableAuthService;
```

Comment:
So looking at these tests, I don't see any changes for the functions that return expression matrices. For the functions that return refs, the only test appears to be that there is some object that exists at that ref, not that the object is as expected.